src/mngt/ocf_mngt_common.c

```diff
@@ -151,6 +151,13 @@ void ocf_mngt_cache_put(ocf_cache_t cache)
 	}
 }
 
+void __set_cleaning_policy(ocf_cache_t cache,
```

Comment:
suggest renaming to ocf_set_cleaning_policy

Reply:
`ocf_set_cleaning_policy` may suggest the function responsible for switching cleaning policy whereas it's only a helper function. Double underscore and the lack of `ocf` prefix are supposed to highlight it.